### ISSUE TITLE
Version 1.3.1 breaks support for node v5

### ISSUE THREAD

**Issue by mearns** [mearns/extrinsic-promises#5]
Mocha tests fail in node v5, starting in version 1.3.1 with the following error:

```
.../extrinsic-promises/node_modules/nyc/node_modules/find-up/index.js:5
module.exports = (filename, opts = {}) => {
                                 ^
SyntaxError: Unexpected token =
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:387:25)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/travis/build/mearns/extrinsic-promises/node_modules/nyc/lib/config-util.js:6:16)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
```

**Comment by mearns** [comment]
Won't fix, because the new version of nyc doesn't support v5, but updated to specify v6 as the minimum supported engine.

**Comment by mearns** [comment]
Addressed in [version 2.0.0](https://github.com/mearns/extrinsic-promises/releases/tag/versions%2F2.0.0) by removing support for node v5.
